<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>CallToaster</class>
 <widget class="QWidget" name="CallToaster" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="titleLabel" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="messageLabel" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QtWengoStyleLabel" name="hangupLabel" >
         <property name="minimumSize" >
          <size>
           <width>28</width>
           <height>56</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>28</width>
           <height>56</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QtWengoStyleLabel" name="callLabel" >
         <property name="minimumSize" >
          <size>
           <width>48</width>
           <height>56</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>48</width>
           <height>56</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="pixmapLabel" >
     <property name="minimumSize" >
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QtWengoStyleLabel</class>
   <extends>QLabel</extends>
   <header>QtWengoStyleLabel_proxy.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
